From 41d6940891b91e27ea5509eda2247811a3ba0c0d Mon Sep 17 00:00:00 2001
From: Joel Stanley <joel@jms.id.au>
Date: Tue, 30 Jul 2019 15:52:05 +0930
Subject: [PATCH] aspeed: Limit bootm memory

This ensures u-boot places the kernel, device tree and initrd at an
address that is accessible by the Linux early boot code.

Without this, when booting with FIT the device tree is placed at the end
of RAM which is not mapped on the AST2600 EVB which contains 2GB of RAM.

Signed-off-by: Joel Stanley <joel@jms.id.au>
---
 include/configs/aspeed-common.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/aspeed-common.h b/include/configs/aspeed-common.h
index 3b345ba46333..eba4e1c79d80 100644
--- a/include/configs/aspeed-common.h
+++ b/include/configs/aspeed-common.h
@@ -41,6 +41,7 @@
 #define CONFIG_SYS_INIT_SP_ADDR \
 	(SYS_INIT_RAM_END - GENERATED_GBL_DATA_SIZE)
 
+#define CONFIG_SYS_BOOTMAPSZ		(256 * 1024 * 1024)
 #define CONFIG_SYS_MALLOC_LEN		(32 << 20)
 
 /*
-- 
2.20.1

